### ISSUE TITLE
Issue with imports - TypeScript < 4

### ISSUE THREAD

**Issue by Davste93** [sequelize/sequelize-auto#498]
![image](https://user-images.githubusercontent.com/39570462/107516238-c480e900-6bb4-11eb-9d2f-7f4ffcb6c484.png)

Is anyone using this with TypeScript?

Not sure why the generated code is doing:

```
import Sequelize, { DataTypes, Model, Optional } from 'sequelize';

static initModel(sequelize: Sequelize.Sequelize): 
```

Instead of:
```
import { Sequelize, DataTypes, Model, Optional } from 'sequelize';

static initModel(sequelize: Sequelize): 
```

I don't know if Sequelize went from default to named exports, but it seems that this is not working as per sequelize 6.5.0.

**Comment by Davste93** [comment]
To anyone looking for a quick workaround:


```
const replace = require('replace-in-file');
...
const auto = new SequelizeAuto(...);
auto.run().then(data => {
  replace.sync({
    ...replaceConfig,
    from: /import Sequelize, \{ DataTypes, Model, Optional \}/g,
    to: 'import { Sequelize, DataTypes, Model, Optional }',
  });

  replace.sync({
    ...replaceConfig,
    from: /Sequelize.Sequelize/g,
    to: 'Sequelize',
  });
});
```

**Comment by steveschmitt** [comment]
You need TypeScript **4.x**.  The syntax
```js
import Sequelize, { DataTypes, Model, Optional } from 'sequelize';
```
imports the Sequelize **namespace**, which is used later in the mixin declarations:
```js
  getCustomer!: Sequelize.BelongsToGetAssociationMixin<Customer>;
```

**Comment by Davste93** [comment]
![image](https://user-images.githubusercontent.com/39570462/109200917-d2845b80-77a9-11eb-9518-9e4cc5d9e9ba.png)

Typescript 4.0.5

**Comment by Davste93** [comment]
> You need TypeScript **4.x**. The syntax
> 
> ```js
> import Sequelize, { DataTypes, Model, Optional } from 'sequelize';
> ```
> 
> imports the Sequelize **namespace**, which is used later in the mixin declarations:
> 
> ```js
>   getCustomer!: Sequelize.BelongsToGetAssociationMixin<Customer>;
> ```

Also am I missing something or could the same thing be achieved with:

> import * as Sequelize, and then Sequelize.BelongsToGetAssociationMixin

**Comment by steveschmitt** [comment]
Hmm, let me try again to reproduce the problem.

**Comment by awcchungster** [comment]
I can replicate this issue:

"typescript": "^4.1.5"

**Comment by awcchungster** [comment]
I found a resolution by changing my tsconfig


  "allowSyntheticDefaultImports": true

**Comment by steveschmitt** [comment]
I think it works for me because I have
```
"esModuleInterop": true,
```
in my **tsconfig.json**.

@Davste93 can you try that?

**Comment by cesar3030** [comment]
> To anyone looking for a quick workaround:
> 
> ```
> const replace = require('replace-in-file');
> ...
> const auto = new SequelizeAuto(...);
> auto.run().then(data => {
>   replace.sync({
>     ...replaceConfig,
>     from: /import Sequelize, \{ DataTypes, Model, Optional \}/g,
>     to: 'import { Sequelize, DataTypes, Model, Optional }',
>   });
> 
>   replace.sync({
>     ...replaceConfig,
>     from: /Sequelize.Sequelize/g,
>     to: 'Sequelize',
>   });
> });
> ```

Thanks for the lead, I had to do a few changes to have it working:

```ts
replace.sync({
  files: './src/models/*.ts',
  from: /import Sequelize, \{ DataTypes, Model, Optional \} from 'sequelize';/g,
  to: "import { DataTypes, Model, Optional } from 'sequelize';\nimport * as Sequelize from 'sequelize';"
});
```

**Comment by steveschmitt** [comment]
Fixed in 0.8.2
